src/backend/launcher.ts

```diff
@@ -352,10 +352,10 @@ async function prepareWineLaunch(
     if (gameSettings.autoInstallDxvk) {
       await DXVK.installRemove(gameSettings, 'dxvk', 'backup')
     }
-    if (gameSettings.autoInstallDxvkNvapi) {
+    if (isLinux && gameSettings.autoInstallDxvkNvapi) {
       await DXVK.installRemove(gameSettings, 'dxvk-nvapi', 'backup')
     }
-    if (gameSettings.autoInstallVkd3d) {
+    if (isLinux && gameSettings.autoInstallVkd3d) {
```

Comment:
IMO, instead of gating these behind `isLinux`, we should make sure the relevant settings values are never `true` on macOS in the first place

Reply:
that's already been done, see the change where the defaults have been changed

Reply:
Thats been done like Etaash said they still come true on macOS and I could not find exactly why. on the app settings it is true only if it is linux but on mac somehow still sets to true the vkd3d. So with the current logic, if vkd3d is true it installs the dxvk as well and it breaks the GPTK.

Reply:
Ah I see, it *was* `true` by default once, so now that it's set to that value, it stays on that (even if we change the default)

I guess this is fine then